build.sh

```
@@ -312,12 +312,18 @@ function print_test_summary() {
 
 function print_test_error_and_exit() {
-    printf " ... ${RED}failed${RESET}\n"
-    if [[ "${DUMP_TEST_ERRORS_STDOUT}" == "yes" ]]; then
-        # Only dump the failed test's output, not the entire accumulated log
-        if [ -f "${CURRENT_TEST_OUTPUT_FILE}" ]; then
-            cat "${CURRENT_TEST_OUTPUT_FILE}"
-        fi
+    local test_name="${1:-unknown}"
+    local test_exit_code="${2:-1}"
+
+    printf " ... ${RED}FAILED (exit code: ${test_exit_code})${RESET}\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)